sdk/identity/azure-identity/test/perf/inc/azure/identity/test/externals/perf.combo.pem

```diff
@@ -0,0 +1,53 @@
+Bag Attributes
```

Comment:
Is this pem file going to trigger security scanner failures?  Could we store this in keyvault and download it as part of the test prep?

Reply:
This is 100% going to trigger credscan.

Reply:
If it doesn't, something is wrong in how credscan is configured :).

Reply:
updated credscan file to allow this cert. having this cert in keyvault does not work, i tried it as keyvault will download only cert backups not certs themselves in the right format

Reply:
How do we ensure that nobody can use this certificate to access any azure services? Is there a way to say "Allow logon but don't allow any service access"?

Reply:
Or is this a bogus certificate that doesn't grant access to any resources?